install/helm/agones/values.yaml

```diff
@@ -164,11 +164,13 @@ agones:
         port: 443
         portName: https
         targetPort: 8443
+        nodePort: 0
```

Comment:
Two thoughts about this.

1. Setting this to 0 seems like a bad default. If we are going to set it to an example value, it seems like it should be a port that would actually work if you just changed the `serviceType` above.
2. This file doesn't really have many comments, but it seems like if we are going to include a value here it is worth adding a comment that it will only be used if the service type is set to NodePort.

Reply:
Thank you for the feedback.

1. I think it is okay to use 0. When I type `kubectl create service nodeport --help` The nodePort parameter is described like this.

```bash
(docker-desktop) ☁  ~  kubectl create service nodeport --help
--node-port=0: Port used to expose the service on each node in a cluster.
```

I think this is because v1.ServiceSpec.ServicePort is using int32.

https://github.com/kubernetes/api/blob/b8c40e080bc5e830097df540d4ef804034cb5bdb/core/v1/types.go#L4536

I found some examples that using `NodePort: 0` in the test code of `kubectl` with `expectErr: false`.

https://github.com/kubernetes/kubectl/blob/c26d922e2ead95fb4850c73c3c41027da40f3e0e/pkg/cmd/create/create_service_test.go#L204-L221

It is a good idea to use an example NodePort with the default range of 30000-32767. However, it could make an error if the port is already taken by another Service.

2. Comments are added right behind the nodePort.